frontend/src/scenes/retention/retentionTableLogic.js

```diff
@@ -24,6 +46,21 @@ export const retentionTableLogic = kea({
                 setProperties: (_, { properties }) => properties,
             },
         ],
+        people: {
+            updatePeople: (state, { selectedIndex, people }) => ({
+                ...state,
+                [`${selectedIndex}`]: [...state[selectedIndex], ...people],
+            }),
+        },
+        retention: {
+            updateRetention: (_, { retention }) => retention,
+        },
+        loadingMore: [
+            false,
+            {
+                updateLoadingMore: (state, { status }) => status,
+            },
+        ],
```

Comment:
Isn't this pattern (with `actions.updateLoadingMore`) kind of a Kea no-no? https://kea.js.org/docs/guide/concepts#listeners

Reply:
yup good point. it's unnecessary